ci/build/npm-postinstall.sh

```diff
@@ -41,6 +41,19 @@ main() {
 vscode_yarn() {
   cd lib/vscode
   yarn --production --frozen-lockfile
+
+  # VS Code needs a node_modules.asar but that's just a duplicate of stuff we
```

Comment:
Let's document what `asar` means/why VS code expects it.

Reply:
Also should document that this is a copy of the function `symlink_asar` in `lib.sh`